docs/naps/4-async-slicing.md

```diff
@@ -223,6 +223,28 @@ It's important to understand what state is currently used for slicing in napari.
     - `_max_points_thumbnail`: `int`, if more points than this in slice, randomly sample them
     - `_selected_view`: `list[int]`, intersection of `_selected_data` and `_indices_view`, could be a cached property
 
+- `Shapes`
+    - `_data_view`: `ShapeList`, container around shape data
+    - `_feature_table`: `_FeatureTable`, dataframe-like features table, scales to number of shapes
+
+	- `ShapeList`
+		- `_slice_key`: `List(int)`, current slice key
+		- `_mesh`: `Mesh`, container to store meshes from shapes
+		- `shapes`: `List(Shape)`, list of underlying shapes
+		- `displayed_vertices`, `List`
```

Comment:
Is this something like `self._mesh.vertices[self.displayed_index]? Would be good to add a short description.